config.yaml

```diff
@@ -84,7 +84,7 @@ options:
           your ceph cluster).
     source:
         type: string
-        default:
+        default: ''
```

Comment:
Is there a reason you've chosen to use empty strings instead of NULL?

Reply:
I've used an empty string ever since i read https://bugs.launchpad.net/pyjuju/+bug/1101607.  i'll fire up a local test to see if this new-fangled NULL thing is actually a thing.

Reply:
We've come a long way since 2013 (and this charm has had those configs for a while). I just want to make sure there wasn't any sideffects of testing for `None` vs `""`

Reply:
Interestingly, if i use 'null' (or variants thereof), proof still hollers:
```bash
$ charm proof
...
I: config.yaml: option key has no default value
```

I did a quick test with `source: ''` and `key: null`, and it seems there *is* a difference:
```bash
$ juju run --unit mysql/2 'config-get -a' | grep -E 'key|source'
key: null
source: ""
```

Further looking into charmhelpers shows `''` vs `` may be a problem:
https://github.com/juju/charm-helpers/blob/master/charmhelpers/fetch/ubuntu.py#L412

@marcoceppi thanks for shaking my confidence down to null.  I'm gonna revert these config changes back to the way they were and let the INFO wars live on.